source/grid/grid_out.cc

```diff
@@ -1584,7 +1584,8 @@ GridOut::write_svg(const Triangulation<2, 2> &tria, std::ostream &out) const
   // (, and level subdomain id).
   for (const auto &cell : tria.cell_iterators())
     {
-      for (unsigned int vertex_index = 0; vertex_index < 4; vertex_index++)
+      for (unsigned int vertex_index = 0; vertex_index < cell->n_vertices();
+           vertex_index++)
```

Comment:
```suggestion
           ++vertex_index)
```
clang-tidy will catch this so we should just fix it now.